.cirrus.yml

```diff
@@ -129,7 +129,8 @@ cross_build_task:
 
     script:
         - brew update
-        - brew install go
+        - brew install go@1.18
+        - export PATH="/usr/local/opt/go@1.18/bin:$PATH"
```

Comment:
The note in https://cirrus-ci.org/guide/writing-tasks/#script-instruction seems relevant here.

Reply:
Oh right, good catch `Note: Each script instruction is executed in a newly created process, therefore environment variables are not preserved between them.`  I was afraid about including the (old)`$PATH` value at the end, since it's not set anywhere in the YAML.  Maybe Cirrus will pass the literal `$PATH` through so it will work?  Let's see...